webapp/components/interop.js

```diff
@@ -172,10 +174,9 @@ class InteropDataManager {
 
         let summaryScore = testScore / numFocusAreas;
```

Comment:
This suggestion goes with the next one, if we'll first compute `testScore`, `investigationScore` and `investigationWeight`:

```suggestion
       testScore /= numFocusAreas;
```